qm-windowmanager/etc/systemd/system/wayland-session.service

```diff
@@ -0,0 +1,37 @@
+[Unit]
+Description=Wayland Session Creation Handling
+After=systemd-user-sessions.service
+
+[Service]
+Type=simple
+Environment=XDG_SESSION_TYPE=wayland
+UnsetEnvironment=TERM
+ExecStart=/bin/sleep infinity
+Restart=no
+
+# Run the session as root (required by PAMName)
+User=0
+Group=0
+
+# Set up a full user session for the user, required by Wayland.
+PAMName=login
```

Comment:
This should be wayland. That is probably what is making the session start to fail.